src/applications/vaos/appointment-list/index.jsx

```diff
@@ -29,13 +30,14 @@ function AppointmentListSection() {
         {isInCCPilot && <Redirect from="/pending" to="/referrals-requests" />}
 
         <Route path="/pending" component={AppointmentsPage} />
+        {isInCCPilot &&
```

Comment:
What did this fix?

Reply:
Because if `isInCCPilot` is true it would never reach the EPS page if it was on the line below. This checks for `isInCCPilot` and `eps` and if that is false then it will still render the `ReferralsAndRequests` if `isInCCPilot` is still true but `eps` is false.